### ISSUE TITLE
Unhandled Stylet.Xaml.ActionNotSetException

### ISSUE THREAD

**Issue by Zillode** [canton7/SyncTrayzor#319]
I got the following trace when clicking on the taskbar icon. Relevant settings:
- Synctrayzor 1.1.12
- Syncthing v0.14.9
- My Syncthing home folder was not mounted.
- Synctrayzor starts at boot, minimized

```
An unhandled exception of type 'Stylet.Xaml.ActionNotSetException' occurred in Stylet.dll
Additional information: View.ActionTarget not set on control Hardcodet.Wpf.TaskbarNotification.TaskbarIcon (method DoubleClick). This probably means the control hasn't inherited it from a parent, e.g. because a
ContextMenu or Popup sits in the visual tree. You will need so set 's:View.ActionTarget' explicitly. See the wiki section "Actions" for more details.
```

**Comment by canton7** [comment]
Thanks. Can you post the entire error message you were given, including the stack trace, please?

**Comment by canton7** [comment]
Actually, that's probably not required - I can figure out where this was triggered from.

**Comment by canton7** [comment]
This should now be fixed: with v1.1.26, the tray icon is hidden until start has reached the point where it can handle interactions
